generators/reference_chunking/noun_splitter/README.md

```diff
@@ -0,0 +1 @@
+Uses spaCy to find nouns in a texts and returns a list of all nouns in a text. This can be used to create an embedding list or use the nouns as pointers to retrieve the original text in a RAG use-case.
```

Comment:
typo `find nouns in a text`,
I find `and returns them in a list` more concise

> This can be used to create an embedding list or use the nouns as pointers to retrieve the original text in a RAG use-case.

I suggest rephrasing this into
`In RAG use-cases it can be a successful strategy to conduct a similarity search only on the nouns. This can be done in refinery by using this brick to create an embedding list.`

Reply:
Added this, thanks! 👍